### PR TITLE
unblock generated docs in https://github.com/kubernetes/kubernetes/pull/43443

### PR DESCRIPTION
**Pull request by BenTheElder**
updates block-path.yaml to sync with [changes for kubefed](https://github.com/kubernetes/kubernetes/pull/43443)

/cc @chenopis

### CODE REVIEW

File: mungegithub/block-path.yaml [hunk 1]
```diff
@@ -16,3 +16,9 @@ doNotBlockRegexp:
   - ^docs/admin/high-availability/.*\.(yaml)$
   - ^docs/man/man1/kubectl.*\.1$
   - ^docs/user-guide/kubectl/kubectl.*\.md$
+  - ^docs/admin/kubefed.md$
```
Comment (ixdy) [inline mungegithub/block-path.yaml]: nit: sort this?

**Comment by krzyzacy** [comment]
can it be a glob? or only those 6 md files?

**Comment by BenTheElder** [comment]
they come from
https://github.com/irfanurrehman/kubernetes/blob/e825ca53bd145e71c43fd2e7fc0c740c323f087b/docs/.generated_docs
for files that would otherwise be blocked (as far as I can tell)

File: mungegithub/block-path.yaml (hunk 1 above)
Comment (BenTheElder) [inline mungegithub/block-path.yaml]: fixing..

Comment (BenTheElder) [inline mungegithub/block-path.yaml]: sorted

**Comment by chenopis** [comment]
IMHO, I don't think a glob is necessary. This is just to unblock that particular PR. The long term fix is to overhaul the autogenerated docs and have someone own that.

**Comment by krzyzacy** [comment]
:+1:

**Comment by BenTheElder** [comment]
@cjwagner when you do that next re-deploy can you also include this?

**Pull request merged**